src/unitxt/loaders.py

```diff
@@ -474,3 +474,21 @@ def process(self):
         return FixedFusion(
             origins=self.sources, max_instances_per_origin=self.get_limit()
         ).process()
+
+
+class LoadDirectory(Loader):
```

Comment:
I think it should be either LoadDictionary or LoadFromConstants.

Please also add a documentation string with example of how to use it, so will be appear in the auto generated documentation.